### ISSUE TITLE
Proposed tweaks to ambpdb and cpptraj PDB output

### ISSUE THREAD

**Issue by drroe** [Amber-MD/cpptraj#375]
Originally from @dacase:

1.  With the -bres option, convert "WAT" to "HOH" as a residue name.  The pdb standard kind of supports both, but many crystallographic programs (and real PDB entries) use HOH.  [A little dangerous, since some workflows may expect WAT, so I suppose this could be controlled by a separate flag; but I actually don't think that should be necessary.]

2.  Right now, ambpdb does not emit a TER card if the residue is WAT; it would be helpful if it *did* emit a TER card between a WAT and an non-WAT residue.

**Comment by drroe** [comment]
> Right now, ambpdb does not emit a TER card if the residue is WAT; it would be helpful if it did emit a TER card between a WAT and an non-WAT residue.

@dacase I thought this was the current behavior in `ambpdb`. If I do a quick test with some files available in the cpptraj test directory (trpzip2 peptide in water):
```
ambpdb -p tz2.truncoct.parm7 -c tz2.truncoct.nc > temp.pdb
```
I get (only showing relevant residues for clarity):
```
ATOM    217  O   LYS    12       2.530   4.759  -8.048  1.00  0.00           O
ATOM    218  N   NHE    13       3.690   6.391  -7.216  1.00  0.00           N
ATOM    219  HN1 NHE    13       4.562   6.844  -6.978  1.00  0.00           H
ATOM    220  HN2 NHE    13       2.780   6.591  -6.826  1.00  0.00           H
TER     221      NHE    13
ATOM    221  O   WAT    14       5.073  10.420   6.330  1.00  0.00           O
ATOM    222  H1  WAT    14       5.056  11.200   5.776  1.00  0.00           H
ATOM    223  H2  WAT    14       4.902  10.749   7.212  1.00  0.00           H
ATOM    224  O   WAT    15       6.303  -7.766 -13.643  1.00  0.00           O
```
Do you have an example where this is not the behavior?

**Comment by drroe** [comment]
From @dacase 

> The pdb version 3 standard for residue names is that they be right-justified in columns 18-20.  This is not currently done in Traj_PDBfile.cpp.  I'd do it, except that something like rname[0] is not an lvalue (can't be modified).

> Could you look into this? Remember that although rname is treated like a 4-character string, the right-justification is in the first three characters of this.

**Comment by drroe** [comment]
Addressed by #398
